spanner/api/Spanner.Samples/DeleteUsingDmlReturningAsync.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_delete_dml_returning]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class DeleteUsingDmlReturningAsyncSample
+{
+    public async Task<List<string>> DeleteUsingDmlReturningAsync(string projectId, string instanceId, string databaseId)
```

Comment:
Hmm.. I was thinking if we should have returned `Task<List<Singer>>` instead of `Task<List<string>>` here and in other samples. We could have asserted on other columns. However, this should be fine as well.